src/test/java/yanagishima/config/YanagishimaConfigTest.java

```diff
@@ -1,60 +0,0 @@
-package yanagishima.config;
-
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-
-import java.util.List;
-import java.util.Properties;
-
-import org.junit.jupiter.api.Test;
-
-class YanagishimaConfigTest {
```

Comment:
Isn't this test necessary?

Reply:
I think it's no longer needed. The coverage is low and mostly meaningless.